bindings/Python/py11glue.cpp

```diff
@@ -229,6 +229,15 @@ PYBIND11_MODULE(adios2, m)
                  const bool opBool = adios ? true : false;
                  return opBool;
              })
+        .def(pybind11::init<const bool>(),
+             "adios2 module starting point "
+             "non-MPI, constructs an ADIOS class "
+             "object",
+             pybind11::arg("debugMode") = true)
```

Comment:
Even though the debug parameter is effectively ignored, should this default to `false` instead of `true`?

Reply:
I guest it doesn't really matter since it's not actually used.

Reply:
I thought about that but decided to keep the change to a minimum.  The old code defaulted to `true` already.